src/types/lib/types.vcxproj.filters

```diff
@@ -60,9 +60,6 @@
     <ClCompile Include="..\UTF8OutPipeReader.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\WindowUiaProvider.cpp">

```

Comment:
I don't know what this is, there is no `WindowUiaProvider.cpp` file in `src/types`. The file is in `src/interactivity/win32`. I think Visual Studio removed it automatically? Should I remove this?

Reply:
This is fine to remove. We occasionally miss updates to the `filters` files, and VS is more than happy to change them where necessary.